File: ComplementApp.API/Dtos/ActividadGeneralPrincipalDto.cs
using System.Collections.Generic;
using ComplementApp.API.Models;

namespace ComplementApp.API.Dtos
{
    public class ActividadPrincipalDto
    {
        public ICollection<ActividadGeneral> ListaActividadGeneral { get; set; }
        public ICollection<ActividadEspecifica> ListaActividadEspecifica { get; set; }
    }
}

File: ComplementApp-SPA/src/app/_dto/actividadGeneralPrincipalDto.ts
import { ActividadEspecifica } from '../_models/actividadEspecifica';
import { ActividadGeneral } from '../_models/actividadGeneral';

export class ActividadGeneralPrincipalDto {
    listaActividadGeneral: ActividadGeneral[];
    listaActividadEspecifica: ActividadEspecifica[];

}
